web/html/src/manager/content-management/shared/business/filters.enum.js

```diff
@@ -42,7 +48,8 @@ function findByKey(key: string): FilterOptionType {
 function getFilterDescription (filter: Object): string {
   const matcherDescription = {
     equals: " matching ",
-    contains: " containing "
+    contains: " containing ",
+    greatereq: " greater or equal "
   }
 
   return `${filter.name}: deny ${filter.entityType} ${matcherDescription[filter.matcher] || ""} ${filter.criteriaValue} (${filter.criteriaKey})`;
```

Comment:
@lneves12 this line produce a text like this in the UI:

```
August: deny erratum greater or equal 2019-08-01T00:00:00+02:00 (issue_date)
```

Any idea how we can show the date in a better way?

Reply:
Hum we probably need to define a specific formater for each filter as we don’t have a way of guessing the type of the criteriavalue. I can handle it with the refactor I have to do in the timezones handling

Reply:
If we want to format it in a prettier way with the server timezone until we add the "moment-timezone" library we have to do it in the server.

Reply:
We can work on this also later